src/Options.php

```diff
@@ -287,6 +287,18 @@ public function setInAppExcludedPaths(array $paths): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    public function getInAppIncludedPaths(): array
+    {
+        return $this->options['in_app_include'];
+    }
+
+    public function setInAppIncludedPaths(array $paths): void
```

Comment:
Missing the PHPDoc block. Please document both what the function does and the argument with a description

Reply:
To clarify, I would like the PHPDoc to document that the `$paths` argument accepts a list of `string`